### PR TITLE
Task 3 refactored

### PR DESCRIPTION
**Pull request by Szpytma**
Task 3 completed and refactored

### CODE REVIEW

File: controllers/pathDoesExistError.controller.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+exports.errorHandler = (req, res) => {
```
Comment (roisinoreilly) [inline controllers/pathDoesExistError.controller.js:1]: nice job extracting this into its own controller

File: __tests__/app.test.js [hunk 2]
```diff
@@ -7,24 +7,44 @@ const connection = require("../db");
...
+        expect(error).toEqual("This Path does not exist");
+      });
+  });
+  it("404: responds with a message { error: 'This Path does not exist'}", () => {
+    return request(app)
+      .get("/")
+      .expect(404)
+      .then(({ body }) => {
+        const { error } = body;
+        expect(error).toEqual("This Path does not exist");
```
Comment (roisinoreilly) [inline __tests__/app.test.js:26]: you could use .toBe here and make your tests stricter

**Review by roisinoreilly — APPROVED** [review]
nicely refactored, merge when ready :)

File: __tests__/app.test.js (hunk 2 above)
Comment (Szpytma) [reply __tests__/app.test.js:26]: I have changed it, I will push it with the next completed ticket

**Pull request merged**